openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -7,7 +7,8 @@
 
 from django.conf import settings
 from django.db import models, transaction
-from django.db.models.fields import BooleanField, DateTimeField, DecimalField, TextField, FloatField, IntegerField
```

Comment:
Remove the change in this line to reduce git diff.

Reply:
done